### PR TITLE
Create tictactoe.py

### PR DESCRIPTION
**Pull request by PythonSerious**
I've re-created tic-tac-toe with an easy to read format.

- [ ] New feature
- [ ] Fix bug
- [ ] Typo
- [x] Other

## Checks
- [ ] Did you use the black formatter?
- [ ] Does this requires the users to change their code?
- [x] Did you test?
- [ ] Have you updated the docs?
- [x] Can you listen to our requests?
- [ ] Did you use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

File: examples/games/tictactoe.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+@PythonSerious - 2021
+https://github.com/PythonSerious
+"""
+import discord
+from discord.ext.commands import command, Cog
+from discord_components import DiscordComponents, Button, ButtonStyle, InteractionType
+import asyncio
+
+
+class tictactoe(Cog):
```
Comment (kiki7000) [inline examples/games/tictactoe.py:11]: Could you use PascalCase for class name?

File: examples/games/tictactoe.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    def __init__(self, bot):
+        self.bot = bot
+
+
+    @command(aliases=['tictactoe'])
+    async def ttt(self, ctx, member: discord.Member):
```
Comment (kiki7000) [inline examples/games/tictactoe.py:17]: Use the pascal case for the class name and name this `tictactoe` without aliases

File: examples/games/tictactoe.py [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                    return True
+                else:
+
+                    return False
+            else:
+
+                return False
+
+
+        m = await ctx.send(embed=embed, components=acceptdenycomps, content=member.mention)
```
Comment (kiki7000) [inline examples/games/tictactoe.py:96]: Could you put this before function defining?

File: examples/games/tictactoe.py [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+                    )
+                    return
+
+
+
+
+
+def setup(bot):
+    print('Tictactoe - by python#0001 has loaded!')
+    DiscordComponents(bot)
```
Comment (kiki7000) [inline examples/games/tictactoe.py:221]: A comment here

File: examples/games/tictactoe.py [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+        greenstatus = True
+        # True = green False = red
+        def greensturncheck(res):
+            return res.user.id == ctx.author.id and res.channel.id == ctx.channel.id and res.message.id == m.id
+        def redsturncheck(res):
+            return res.user.id == member.id and res.channel.id == ctx.channel.id and res.message.id == m.id
+        while accept:
+            if greenstatus:
+                await m.edit(embed=greensturnembed, components=board)
+                try:
```
Comment (kiki7000) [inline examples/games/tictactoe.py:150]: Just put a `try ~ except` on the `await bot.wait_for`

**Review by kiki7000 — COMMENTED** [review]
If you do those changes, I'll do the rest

File: examples/games/tictactoe.py [hunk 6]
```diff
@@ -8,13 +8,13 @@
 import asyncio
 
 
-class tictactoe(Cog):
+class PascalCase(Cog):
```
Comment (kiki7000) [inline examples/games/tictactoe.py:11]: Oh I didn't mean that. What I was trying to say is change the class name to `TicTacToe`

**Comment by kiki7000** [comment]
Could you use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)? It would be easier to know what was changed

**Comment by PythonSerious** [comment]
> Could you use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)? It would be easier to know what was changed

No because i dont know how to do that, but i changed all of what you asked.

**Comment by kiki7000** [comment]
Please use it in other PRs, or it will not be merged

**Review by kiki7000 — APPROVED** [review]

**Pull request merged**